ref(integrations): Mention metric alert actions (#5035)
getsentry/sentry-docs: src/docs/product/integrations/notification-incidents/slack/index.mdx

Slack defaults to let any workspace member authorize apps, but they may have to request access. See this [Slack help article](https://get.slack.help/hc/en-us/articles/202035138-Add-an-app-to-your-workspace) for more details.

1.  In Sentry, navigate to **Settings > Integrations > Slack**.

2.  Click "Add Workspace".

    ![](slack-overview-details-page.png)

> 3.  Toggle to the Slack workspace to which you want to connect using the dropdown menu in the upper right corner of the authentication window, then select **Allow**. Repeat this process if you are connecting to multiple workspaces.

Reviewer: ```suggestion
3.  Toggle to the Slack workspace to which you want to connect using the dropdown menu in the upper right corner of the authentication window. Then select "Allow". Repeat this process if you are connecting to multiple workspaces.
```

    ![](slack-allow-install.png)

> 4.  Your integration details page will refresh and show the Slack workspace you just added.

Reviewer: Not a step

```suggestion
Your integration details page will refresh and show the Slack workspace you just added.
```

### Alert Rules

> Navigate to **Alerts > Create Alert**. From here, you can configure when notifications are sent to your Slack workspace(s).

Reviewer: ```suggestion
Set up an alert rule by going to **Alerts** and clicking "Create Alert*". From here, you can configure alerts to route notifications to your Slack workspace(s).
```

> You can select Slack as either an [issue alert](/product/alerts/alert-types/#issue-alerts) action:

Reviewer: ```suggestion
In [issue alerts](/product/alerts/alert-types/#issue-alerts), select "Send a PagerDuty notification" in the actions dropdown and then select your workspace and channel:
```

![](slack_issue_alert.png)

or a [metric alert](/product/alerts/alert-types/#metric-alerts) action.

![](slack_metric_alert.png)

> Once you receive a Slack notification for an issue alert, you can use the Resolve, Ignore, or Assign buttons to update the Issue in Sentry.

Reviewer: ```suggestion
Once you receive a Slack notification for an issue alert, you can use the "Resolve", "Ignore", or "Assign" buttons to update the Issue in Sentry.
```
